### PR TITLE
Mp 1532 add constraints

### PR DESCRIPTION
**Pull request by kuryshev**

### CODE REVIEW

File: model-generator/src/test/resources/expected/kotlin/ConstraintRequiredFieldRootDto.txt [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import kotlin.Int
+import kotlin.String
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.PROPERTY)
+@JsonSubTypes(
+    JsonSubTypes.Type(name = "T1", value = RequiredFieldChild1Dto::class),
+    JsonSubTypes.Type(name = "T2", value = RequiredFieldChild2Dto::class)
+)
+sealed class ConstraintRequiredFieldRootDto {
+    abstract val id : String
```
Comment (yakischik) [inline model-generator/src/test/resources/expected/kotlin/ConstraintRequiredFieldRootDto.txt:16]: Constraint for abstract field

File: model-generator/src/test/resources/expected/kotlin/ConstraintSinglePrimitivesItemDto.txt [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+data class ConstraintSinglePrimitivesItemDto (
+    @field:Min(1)
+    @field:Max(111)
+    val id : Long = 12,
+    val name : String = "noname",
+    @field:Max(100)
+    val index : Int,
+    @field:Min(1)
+    val count : Int,
```
Comment (yakischik) [inline model-generator/src/test/resources/expected/kotlin/ConstraintSinglePrimitivesItemDto.txt:21]: Remove unused fields

File: model-generator/src/test/resources/expected/kotlin/ConstraintRequiredFieldRootDto.txt (hunk 1 above)
Comment (kuryshev) [reply model-generator/src/test/resources/expected/kotlin/ConstraintRequiredFieldRootDto.txt:16]: done

File: model-generator/src/test/resources/expected/kotlin/ConstraintSinglePrimitivesItemDto.txt (hunk 2 above)
Comment (kuryshev) [reply model-generator/src/test/resources/expected/kotlin/ConstraintSinglePrimitivesItemDto.txt]: done

**Review by yakischik — APPROVED** [review]

**Pull request merged**